map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt

```diff
@@ -25,9 +28,18 @@ class GlideMapTileProvider(private val context: Context,
 
                 val path = "http://aic-mobile-tours.artic.edu/sites/default/files/floor-maps/tiles/floor$floor/zoom$zoom/tiles-$tileNumber.png"
                 return try {
+                    val bitmap = Glide.with(context).asBitmap().load(path).submit(TILE_SIZE.toInt(), TILE_SIZE.toInt()).get()
```

Comment:
Add comment above this line so it's super clear how much of this code is temporary. Say something like `PNG decoding triggers crashes on some Android 8.1+ devices`.

Reply:
done